BUILD.gn

```diff
@@ -736,6 +737,8 @@ if (is_mac) {
       sources = [
         "$root_out_dir/egl_intermediates/libswiftshader_libEGL.dylib",
         "$root_out_dir/egl_intermediates/libswiftshader_libGLESv2.dylib",
+        "$root_out_dir/vk_intermediates/libvk_swiftshader.dylib",
+        "$root_out_dir/vk_intermediates/vk_swiftshader_icd.json",
```

Comment:
Should also add the `swiftshader_vulkan_symbols` target.

Reply:
@deepak What about the `angle_vulkan_symbols`?:  https://chromium-review.googlesource.com/c/chromium/src/+/2056931/29/chrome/BUILD.gn#1650 or do we not use them?

Reply:
Angle backend is only used with `--use-gl=angle` , not by default yet on linux and macOS. So we can ship those symbols as chrome does. Right now chrome seems to ship it for linux, so we should do that.

Reply:
Looks like the linux symbol changes were unnecessary:
https://source.chromium.org/chromium/chromium/src/+/abafaff8b0398c70bd12d1dcc7d5b100fd426a19